### ISSUE TITLE
Make 'SDK' highlight flexible

### ISSUE THREAD

**Issue by ewillink** [eclipse-justj/justj#24]
The JustJ indexes show some features in bold. This is not explained to the user. Debugging reveals that there is a hardwired pattern match to `".*\\.sdk([_.-]feature)?\\.feature\\.group"`. This was plausible in pre-Tycho days when SDK meant with-sources. But now that we can have with/without Developer Resources for all plugins/features, SDK is a misnomer and so the hardwired match may fail to highlight rationally-named primary features.

Suggest add a -bold-iu-pattern to parameterize the hardwired pattern.

**Comment by merks** [comment]
I like that idea.   Note that this thing started out as a copy of infrastructure that was used only for EMF and then was generalized from that basis, as I used it for more and more projects...

**Comment by ewillink** [comment]
Yes and very useful it is. (Though when I searched for it in EMF I couldn't find it. Maybe it had already moved.)
